lib/skunk/commands/default.rb

```diff
@@ -27,7 +25,7 @@ def initialize(options)
       #
       # @return [Skunk::Command::StatusReporter]
       def execute
-        RubyCritic::Config.formats = []
+        RubyCritic::Config.formats = [:json]
```

Comment:
This is a temporal since we currently only support the `:json` generator; it's defaulted to it. but this will be removed eventually.